src/WebJobs.Script.WebHost/Middleware/FunctionInvocationMiddleware.cs

```diff
@@ -128,7 +128,8 @@ private async Task<IActionResult> GetResultAsync(HttpContext context, IFunctionE
                 using (logger.BeginScope(scopeState))
                 {
                     var applicationLifetime = context.RequestServices.GetService<IApplicationLifetime>();
-                    CancellationToken cancellationToken = applicationLifetime != null ? applicationLifetime.ApplicationStopping : CancellationToken.None;
+                    CancellationToken cancellationToken = context.HttpContext.RequestAborted ?? CancellationToken.None;
```

Comment:
Is `RequestAborted` nullable? If not, you don't need to null-coalesce, as it will default to `CancellationToken.None` as it is a struct.

Reply:
Good call out; will double check and update as necessary